geode-core/src/distributedTest/java/org/apache/geode/internal/cache/RemoteTransactionDUnitTest.java

```diff
@@ -3636,9 +3542,7 @@ public Object call() {
         Region cust = getCache().getRegion(D_REFERENCE);
         OneUpdateCacheListener rat =
             (OneUpdateCacheListener) cust.getAttributes().getCacheListener();
-        if (!rat.getSuccess()) {
-          fail("The OneUpdateCacheListener didnt get an update");
-        }
+        assertThat(rat.getSuccess()).isTrue();
```

Comment:
The failure description from the original test is lost here. It would be good to add it back with `as()`